resources/js/components/LoginPage.vue

```diff
@@ -117,13 +117,18 @@ export default {
     }
   },
   methods: {
-    login() {
+    submit() {
       // We've seen double submits of the login form, leading to 419 errors.  Prevent the user submitting twice by
-      // double-clicking.
+      // double-clicking by ignoring submits within 1 second of the last submit.
```

Comment:
Any harm in making this time a bit longer?  e.g. 5 seconds.  

In my experience of having this happen when logging in via mobile, it's when browser autosubmit happens and you have to wait a while for the login to complete.  You press it again because you didn't realise the autocomplete also autosubmit.   If the login occurs in less than a second, you probably won't actually press the button again.